pkg/internal/sdk/template/brc29/brc29_template_test.go

```diff
@@ -0,0 +1,189 @@
+package brc29_test
+
+import (
+	"testing"
+
+	"github.com/4chain-ag/go-wallet-toolbox/pkg/internal/sdk"
+	"github.com/4chain-ag/go-wallet-toolbox/pkg/internal/sdk/template/brc29"
+	"github.com/bsv-blockchain/go-sdk/chainhash"
+	ec "github.com/bsv-blockchain/go-sdk/primitives/ec"
+	"github.com/bsv-blockchain/go-sdk/script/interpreter"
+	"github.com/bsv-blockchain/go-sdk/transaction"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBRC29TemplateLock(t *testing.T) {
+	t.Run("should lock with P2PKH and BRC29 calculated address", func(t *testing.T) {
+		lockingScript, err := brc29.Lock(senderPrivateKeyHex, keyID, recipientPublicKeyHex)
```

Comment:
```suggestion
                // Given
		lockingScript, err := brc29.Lock(senderPrivateKeyHex, keyID, recipientPublicKeyHex)
```

and same suggestions as i pointed out in above test file